Rakefile

```diff
@@ -11,7 +11,7 @@ Hoe.spec "minitest" do
 
   license "MIT"
 
-  require_ruby_version [">= 2.2", "< 3.1"]
+  require_ruby_version [">= 2.2", "< 3.2"]
```

Comment:
IMHO it would be better to simply get rid of the upper bound. There's no reason to assume future version will break minitest, and this kind of restriction make testing ruby-head a huge pain.

Reply:
I haven't looked to see if Ruby is following semantic versioning, but I believe that several several std-lib gems are.

Hence, since Minitest is working with 3.0, the following would certainly be appropriate:
```ruby
require_ruby_version [">= 2.2", "< 4.0"]
```